script.audio.profiles/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.audio.profiles" name="Audio Profiles" version="1.4.2~beta5" provider-name="pkscout, notoco, Regss">
```

Comment:
beta?

Reply:
Crap. I didn't push the final merge from my desktop to Github before downloading the zip. I've fixed that, but there will be a few other new lines of code to check when push the update.